website/src/components/explorer/map.jsx

```diff
@@ -0,0 +1,223 @@
+            maxX = coord[0];
+          }
+          if (coord[1] < minY) {
+            minY = coord[1];
+          } else if (coord[1] > maxY) {
+            maxY = coord[1];
+          }
+        }
+      }
+
+      if (
+        Number.isFinite(minX) &&
+        Number.isFinite(minY) &&
+        Number.isFinite(maxX) &&
+        Number.isFinite(maxY) &&
+        width > 1 &&
+        height > 1
+      ) {
+        const { latitude, longitude, zoom } = viewport.fitBounds(
+          [
+            [minX, minY],
+            [maxX, maxY],
+          ],
+          { padding: 96 },
+        );
+
+        setCurrentInitialViewState({
+          latitude,
+          longitude,
+          zoom,
+          transitionInterpolator: new FlyToInterpolator(),
+          transitionDuration: 1600,
+        });
+      }
+    }
+  }, [userInput, userValidHex, deckLoaded]);
+
+  const layers = userValidHex
+    ? [
+        new H3HexagonLayer({
+          id: "userhex",
+          data: userInput.map((hex) => ({ hex })),
+          getHexagon: (d) => d.hex,
+          extruded: false,
+          filled: false,
+          stroked: true,
+          getLineColor: [0, 0, 0],
+          getLineWidth: 3,
+          lineWidthMinPixels: 3,
+          highPrecision: true,
+          pickable: true,
+          filled: true,
+          getFillColor: [0, 0, 0, 30],
+        }),
+      ]
+    : [
+        new H3HexagonLayer({
+          id: "res0",
+          data: res0Cells,
+          getHexagon: (d) => d.hex,
+          extruded: false,
+          filled: false,
+          stroked: true,
+          getLineColor: [0, 0, 0],
+          getLineWidth: 3,
+          lineWidthMinPixels: 3,
+          highPrecision: true,
+        }),
+        new H3HexagonLayer({
+          id: "res1",
+          data: res1Cells,
+          getHexagon: (d) => d.hex,
+          extruded: false,
+          filled: false,
+          stroked: true,
+          getLineColor: [0, 0, 0],
+          getLineWidth: 2,
+          lineWidthMinPixels: 2,
+          highPrecision: true,
+          lineWidthUnits: "pixels",
+          getDashArray: [5, 1],
+          dashJustified: true,
+          dashGapPickable: true,
+          extensions: [new PathStyleExtension({ dash: true })],
+        }),
+        new H3HexagonLayer({
+          id: "res2",
+          data: res2Cells,
+          getHexagon: (d) => d.hex,
+          extruded: false,
+          filled: false,
+          stroked: true,
+          getLineColor: [0, 0, 0],
+          getLineWidth: 1,
+          lineWidthMinPixels: 1,
+          highPrecision: true,
+          lineWidthUnits: "pixels",
+          getDashArray: [5, 5],
+          dashJustified: true,
+          dashGapPickable: true,
+          extensions: [new PathStyleExtension({ dash: true })],
+        }),
+      ];
+
+  const getTooltip = useCallback(({ object }) => {
+    if (object && object.hex) {
+      return {
+        html: `<tt>${object.hex}</tt>`,
```

Comment:
Maybe
```suggestion
        html: `<code>${object.hex}</code>`,
```
? `<tt>` is deprecated